README.md

```diff
@@ -1,35 +1,35 @@
-# ArduPilot Ignition Gazebo plugin
-This is ArduPilot official plugin for Ignition Gazebo.
+# ArduPilot Gazebo Simulator plugin
+This is ArduPilot official plugin for Gazebo Simulator plugin
 It replaces the old Gazebo plugin to bring support of the next gen Gazebo simulator.
 It also brings support for more features :
 - more flexible data exchange between SITL and Ignition with JSON data sharing.
 - more sensors supported.
 - true Simulation lockstepping. It is now possible to use GDB to stop the Ignition time for debugging.
 - Better 3D rendering
 
-The project is composed of an Ignition plugin to connect to ArduPilot SITL (Software In The Loop) and some example models and worlds.
+The project is composed of an Gazebo plugin to connect to ArduPilot SITL (Software In The Loop) and some example models and worlds.
 
 ## Disclaimer :
 The plugin is currently working, but we are working into bringing support for more feature and refine the API.
 
 ## Prerequisites :
-Ignition Fortress is supported on Ubuntu Bionic, Focal and Jammy. If you are running Ubuntu as a virtual machine you will need at least Ubuntu 20.04 (Focal) in order to have the OpenGL support required for the `ogre2` render engine.
+Gazebo Sim Garden is supported on Ubuntu Focal and Jammy. If you are running Ubuntu as a virtual machine you will need at least Ubuntu 20.04 (Focal) in order to have the OpenGL support required for the `ogre2` render engine.
 
-Follow the instructions for a [binary install of ignition fortress](https://ignitionrobotics.org/docs/fortress/install) and verify that ignition gazebo is running correctly.
+Follow the instructions for a [binary install of ignition fortress](https://ignitionrobotics.org/docs/garden/install) and verify that ignition gazebo is running correctly.
```

Comment:
Link is to garden, comment is for fortress. Suggest change comment to garden as well.